### PR TITLE
Add code skleton base on swagger

### PR DESCRIPTION
**Pull request by MakinoAlan**

### CODE REVIEW

File: Service/Controllers/CommentController.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                return Ok();
+            }
+            catch (Exception)
+            {
+                return NotFound();
+            }
+        }
+
+        [HttpPut]
+        [Route("{locationId}")]
```
Comment (mli9502) [inline Service/Controllers/CommentController.cs:49]: For comments, I think we also want to do `/counter/{countryCode}/state/{stateCode}`?
https://github.com/Alying/team_ase_sharp/blob/main/swagger.yaml#L160
But I guess we can update it latter when we actually work on it.

File: Service/Controllers/RecommendationController.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                return Ok();
+            }
+            catch (Exception)
+            {
+                return NotFound();
+            }
+        }
+
+        [HttpGet]
+        [Route("{countryCode}")]
```
Comment (mli9502) [inline Service/Controllers/RecommendationController.cs:36]: [minor]: Do we want to use something like `country/{countryCode}` as route or just use `{countryCode}`?
I'm fine with both ways.

**Review by mli9502 — APPROVED** [review]
lgtm!
Some minor comments. Thanks a lot for the update!

File: Service/Controllers/RecommendationController.cs (hunk 2 above)
Comment (MakinoAlan) [reply Service/Controllers/RecommendationController.cs:36]: Oh I missed it, should have country

File: Service/Controllers/CommentController.cs (hunk 1 above)
Comment (MakinoAlan) [reply Service/Controllers/CommentController.cs:49]: added

File: Service/Controllers/RecommendationController.cs (hunk 2 above)
Comment (MakinoAlan) [reply Service/Controllers/RecommendationController.cs]: changed

**Review by Alying — APPROVED** [review]

**Pull request merged**